homework.cls

```diff
@@ -28,9 +28,11 @@
 \DeclareVoidOption{12pt}{\renewcommand{\@fontsize}{12pt}}
 
 % True/False options. Defaults are False
```

Comment:
These are no longer just true/false options with the addition of the `color` option. Also, I'm wondering if `color` should default to something empty to signify using the default behavior when it's not included.

Reply:
Good points. I will add a clarifying comment, and remove the `default` parameter so it errors if someone calls `\documentclass[boxes,color]{homework}`.